web/src/components/RunDetailLayout/HeaderLeft.tsx

```diff
@@ -7,18 +7,17 @@ import * as S from './RunDetailLayout.styled';
 
 interface IProps {
   name: string;
-  testId: string;
   triggerType: string;
 }
 
-const HeaderLeft = ({name, testId, triggerType}: IProps) => {
+const HeaderLeft = ({name, triggerType}: IProps) => {
   const navigate = useNavigate();
   const {run} = useTestRun();
   const createdTimeAgo = Date.getTimeAgo(run?.createdAt ?? '');
 
   return (
     <S.Section $justifyContent="flex-start">
-      <S.BackIcon data-cy="test-header-back-button" onClick={() => navigate(`/test/${testId}`)} />
+      <S.BackIcon data-cy="test-header-back-button" onClick={() => navigate(-1)} />
```

Comment:
Instead of having a fixed page to go back we just send the user to the prev page